circuits/circom/scripts/verify-onchain.ts

```diff
@@ -26,9 +26,7 @@ const main = async () => {
   })
 
   // FIXME
-  console.log(
-    `${(valid as unknown as boolean) ? '✅ valid proof' : '❌ invalid proof'}`,
-  )
+  console.log(`${(valid as boolean) ? '✅ valid proof' : '❌ invalid proof'}`)
```

Comment:
FYI, that change was to resolve this linting error: 
```
 error  This assertion is unnecessary since it does not change the type of the expression  @typescript-eslint/no-unnecessary-type-assertion
```